lib/r10k/forge/module_release.rb

```diff
@@ -29,8 +39,9 @@ def initialize(full_name, version)
 
         @forge_release = PuppetForge::V3::ModuleRelease.new(@full_name, @version)
 
-        #@download_path = Pathname.new(R10K::Settings[:forge][:module_cache]) + "#{@forge_release.slug}.tgz"
-        #@unpack_path   = Pathname.new(R10K::Settings[:forge][:unpack_tmpdir]) + @forge_release.slug
+
+        @download_path = Pathname.new(Dir.mktmpdir) + (slug + '.tar.gz')
```

Comment:
In the future, yes - rather than go down a rabbit hole of dealing with settings I've chosen to just use a tmpdir to get this merged and then revisit this in a future issue/pull request.